cvat-ui/src/components/email-confirmation-page/email-confirmed.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './styles.scss';
+/**
+ * Component for displaying email confirmation message and then redirecting to the loginpage
+ */
+class EmailConfirmationMessage extends React.Component {
+    componentDidMount() {
+        let counter = 5;
+        const id = setInterval(() => {
+            counter--;
+            document.getElementById('countID').innerHTML = counter;
+
+            if (counter === 0) {
+                try {
+                    document.getElementById('link').click();
+                } catch (error) {
+                    // console.log(error);
+
+                }
+                clearInterval(id);
+            }
+        }, 1000);
+        document.getElementById('link').onclick = function () { clearInterval(id); };
+    }
+    render() {
+        return (
+
+            <section className='ant-layout'>
```

Comment:
Please don't specify any classes here and use Antd components, you can find a lot of examples in the Antd documentation https://ant.design/components/layout/

Reply:
@azhavoro  
I've removed class names and used antd components